commander/src/commands/transaction/create/multisignature.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import {
+	registerMultisignature,
+	utils as transactionUtils,
+} from '@liskhq/lisk-transactions';
+import { isValidFee, isValidNonce } from '@liskhq/lisk-validator';
+import { flags as flagParser } from '@oclif/command';
+
+import BaseCommand from '../../../base';
+import { ValidationError } from '../../../utils/error';
+import { flags as commonFlags } from '../../../utils/flags';
+import { getNetworkIdentifierWithInput } from '../../../utils/network_identifier';
+import { getPassphraseFromPrompt } from '../../../utils/reader';
+
+export default class MultisignatureCommand extends BaseCommand {
+	static args = [
+		{
+			name: 'nonce',
+			required: true,
+			description: 'Nonce of the transaction.',
+		},
+		{
+			name: 'fee',
+			required: true,
+			description: 'Transaction fee in LSK.',
+		},
+	];
+
+	static flags = {
+		...BaseCommand.flags,
+		networkIdentifier: flagParser.string(commonFlags.networkIdentifier),
+		passphrase: flagParser.string(commonFlags.passphrase),
+		'number-of-signatures': flagParser.integer({
+			description:
+				'Number of signatures required to validate transactions from that account',
+		}),
+		'mandatory-key': flagParser.string({
+			multiple: true,
+			description: 'Mandatory public key for multisignature account.',
+		}),
+		'optional-key': flagParser.string({
+			multiple: true,
```

Comment:
same as above

Reply:
Optional keys are itself not required. Either optional or mandatory keys must be given, and that case is covered within lisk-element.